Modules/Loadable/VolumeRendering/MRML/vtkMRMLShaderPropertyNode.h

```diff
@@ -0,0 +1,88 @@
+/// vtkMRMLShaderPropertyNode - MRML node to represent custom volume shader
```

Comment:
All files typically have some copyright text on top. Please see examples in the code and insert one, modified so that you're the original author.

Reply:
Done. I had copied those files from vtkMRMLVolumeProperty*, which doesn't contain a header